invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/search.en.html

```diff
@@ -87,7 +87,9 @@ <h3>Field search</h3>
     >
   </p>
   <p>
-    See the field reference below for the full list of fields you can search.
+    See <a href="https://inveniordm.docs.cern.ch/reference/metadata/#metadata"
```

Comment:
We will need to update the metadata reference, as it's somewhat updated and describes the internal model.